### PR TITLE
Release 10.0.1

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

**Comment by ekohl** [comment]
Travis is failing because it's pulling in newer versions of dependencies. I don't have time to test this before I head out, but it includes the release commit now.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## [10.0.1](https://github.com/theforeman/puppet-foreman_proxy_content/tree/10.0.1) (2020-06-05)
+
+[Full Changelog](https://github.com/theforeman/puppet-foreman_proxy_content/compare/10.0.0...10.0.1)
+
+**Fixed bugs:**
+
+- Fixes [\#29660](https://projects.theforeman.org/issues/29660) - Serve Pulpcore ISO via HTTPS [\#260](https://github.com/theforeman/puppet-foreman_proxy_content/pull/260) ([ekohl](https://github.com/ekohl))
```
Comment (wbclark) [inline CHANGELOG.md:9]: https://github.com/theforeman/puppet-foreman_proxy_content/pull/260 is labeled as an enhancement on the PR, but here it's listed as a bugfix.

I could see the argument for taking either approach, but I think we should be consistent.

**Comment by ehelms** [comment]
I attempted a release but I have old credentials it seems and could not push the update to the forge -- @ekohl sorry but we'll need your help here

**Comment by ehelms** [comment]
Ahh sorry, it was the release-module script assuming I had pass setup. I was able to release this module.

**Pull request merged**